### PR TITLE
Support uartI, allowing for 9 uarts

### PR DESCRIPTION
**Pull request by tridge**
This allows the MatekH743 to use OTG2, allowing for much easier CAN setup
Note that this also adds one more uart on MatekF765, for ESC telem receive

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py [hunk 1]
```diff
@@ -1262,7 +1262,7 @@ def write_UART_config(f):
     num_uarts = len(devlist)
     if 'IOMCU_UART' in config:
         num_uarts -= 1
-    if num_uarts > 8:
+    if num_uarts > 9:
         error("Exceeded max num UARTs of 8 (%u)" % num_uarts)
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py:1266]: shouldn't it be a 9 here?

Comment (tridge) [reply libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py:1266]: thanks! fixed

**Review by khancyr — APPROVED** [review]
LGTM

**Comment by Hwurzburg** [comment]
@tridge not sure that a limit on serial ports is mentioned in the wiki...

**Pull request merged**